### PR TITLE
fix(streaming): acquire conn timeout

### PR DESCRIPTION
**Pull request by chengjoey**
#### What this PR does / why we need it:
fix streaming acquire conn timeout

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=448711&iterationID=-1&type=BUG)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |  fix streaming acquire conn timeout             |
| 🇨🇳 中文    |     修复了streaming获取clickhouse链接大量超时的问题         |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: build/dockerfiles/base/deploy.sh [hunk 1]
```diff
@@ -4,19 +4,14 @@ set -eo pipefail
...
-    docker buildx build --platform linux/${arch} -t ${image} -f Dockerfile --push .
-    # generate Software Bill Of Materials (SBOM)
-    docker sbom ${image} --output sbom-${arch}.txt
-done
+docker buildx build --platform linux/amd64,linux/arm64 -t ${image} -f Dockerfile --push .
 
-for image in ${images[@]}; do
-    echo "action meta: erda-base=$image"
-done
+docker sbom ${image} --output sbom.txt
```
Comment (sfwn) [inline build/dockerfiles/base/deploy.sh:13]: need two sbom

File: cmd/monitor/streaming/bootstrap.yaml [hunk 2]
```diff
@@ -73,10 +75,10 @@ clickhouse:
   username: "${CLICKHOUSE_USERNAME:default}"
   password: "${CLICKHOUSE_PASSWORD:default}"
   conn_open_strategy: "round_robin"
-  conn_max_lifetime: "${CLICKHOUSE_CONN_MAX_LIFETIME:1m}"
-  max_idle_conns: "${CLICKHOUSE_MAX_IDLE_CONNS:10}"
+  conn_max_lifetime: "${CLICKHOUSE_CONN_MAX_LIFETIME:1h}"
+  max_idle_conns: "${CLICKHOUSE_MAX_IDLE_CONNS:195}"
```
Comment (sfwn) [inline cmd/monitor/streaming/bootstrap.yaml:79]: i think idle connections is too much

File: cmd/monitor/streaming/bootstrap.yaml [hunk 3]
```diff
@@ -73,10 +75,10 @@ clickhouse:
...
   max_open_conns: "${CLICKHOUSE_MAX_OPEN_CONNS:200}"
-  dail_timeout: "${CLICKHOUSE_DAIL_TIMEOUT:1s}"
+  dial_timeout: "${CLICKHOUSE_DIAL_TIMEOUT:30s}"
```
Comment (sfwn) [inline cmd/monitor/streaming/bootstrap.yaml:81]: also check envs on updatetext

File: internal/tools/monitor/apm/topology/topology_v2.go [hunk 4]
```diff
@@ -285,6 +285,7 @@ func (topology *provider) GetTopologyV2(orgName string, lang i18n.LanguageCodes,
 
 func (topology *provider) parseToTypologyNodeV2(lang i18n.LanguageCodes, rows driver.Rows, tg *graphTopo) []*relation {
 	allRelations := make([]*relation, 0)
+	defer rows.Close()
```
Comment (sfwn) [inline internal/tools/monitor/apm/topology/topology_v2.go:288]: rows should be closed outside

**Review by sfwn — CHANGES_REQUESTED** [review]
So the real reason is rows-unclosed or bad timeout?

File: build/dockerfiles/base/deploy.sh (hunk 1 above)
Comment (chengjoey) [reply build/dockerfiles/base/deploy.sh]: done

File: cmd/monitor/streaming/bootstrap.yaml (hunk 2 above)
Comment (chengjoey) [reply cmd/monitor/streaming/bootstrap.yaml:79]: idle-conn = maxOpen - 5

File: internal/tools/monitor/apm/topology/topology_v2.go (hunk 4 above)
Comment (chengjoey) [reply internal/tools/monitor/apm/topology/topology_v2.go]: done

**Review by sfwn — APPROVED** [review]

**Pull request merged**